.github/workflows/test_pr_changes.yml

```diff
@@ -97,10 +97,23 @@ jobs:
   trigger_tests_pr:
     needs: detect-changes
     name: PR Tests
+    if: ${{ github.event_name == 'pull_request' }} # run only on PR (not required on develop)
     uses: ./.github/workflows/_reusable_run_tests.yml
     with:
       folders: ${{ join(fromJson(needs.detect-changes.outputs.folders), ',') }}
       notify_secret: WIRE_IOS_CI_WEBHOOK
-      test_dependencies: ${{ github.event_name != 'pull_request' || startsWith(github.base_ref, 'release/cycle') }} # run on merge_queue or release branch PRs
-      run_critical_flows: ${{ github.event_name != 'pull_request' || startsWith(github.base_ref, 'release/cycle') }} # run on merge_queue or release branch PRs
+      test_dependencies: ${{ startsWith(github.base_ref, 'release/cycle') }} # run only on release branch PRs
+      run_critical_flows: ${{ startsWith(github.base_ref, 'release/cycle') }} # run only on release branch PRs
+    secrets: inherit
+    
+  trigger_tests_merge_queue:
```

Comment:
this is duplicated job so we can change the required check to be "Merge Queue Tests"